rest_live/mixins.py

```diff
@@ -85,4 +84,5 @@ def from_scope(cls, viewset_action, scope, view_kwargs, query_params):
         base_request.session = scope.get("session", None)
 
         self.request = self.initialize_request(base_request)
+        self.action = viewset_action  # TODO: custom subscription actions?
```

Comment:
Moved this assignment after the call to `self.initialize_request` because that method will set the `action` to `None` all the time